Aws/Core.hs

```diff
@@ -748,7 +825,7 @@ parseHttpDate s =     p "%a, %d %b %Y %H:%M:%S GMT" s -- rfc1123-date
                   <|> p "%a %b %_d %H:%M:%S %Y" s     -- asctime-date
                   <|> p "%Y-%m-%dT%H:%M:%S%QZ" s      -- iso 8601
                   <|> p "%Y-%m-%dT%H:%M:%S%Q%Z" s     -- iso 8601
-  where p = parseTime defaultTimeLocale
```

Comment:
I've seen those deprecation warnings, but `parseTimeM True` is ugly. :( Oh well, still your change is good.

Reply:
Indeed. :(